Core/src/org/sleuthkit/autopsy/coreutils/Logger.java

```diff
@@ -90,17 +90,21 @@ private static FileHandler createFileHandlerWithTraces(String logDirectory) {
             fileHandler.setFormatter(new Formatter() {
                 @Override
                 public String format(LogRecord record) {
-                    if (record.getThrown() != null) {
+                    Throwable thrown = record.getThrown();
+                    if (thrown != null) {
```

Comment:
I don't know if the newline is required. As long as I get my full stack
traces with no white space between log entries, I'm a happy camper!

On Thu, Mar 31, 2016 at 12:52 PM, Oliver Spohngellert <
notifications@github.com> wrote:

> In Core/src/org/sleuthkit/autopsy/coreutils/Logger.java
> <https://github.com/sleuthkit/autopsy/pull/2054#discussion_r58088335>:
>
> > @@ -90,17 +90,21 @@ private static FileHandler createFileHandlerWithTraces(String logDirectory) {
> >              fileHandler.setFormatter(new Formatter() {
> >                  @Override
> >                  public String format(LogRecord record) {
> > -                    if (record.getThrown() != null) {
> > +                    Throwable thrown = record.getThrown();
> > +                    if (thrown != null) {
>
> Unsure, will consult Richard on that
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/sleuthkit/autopsy/pull/2054/files/d4e77f6687732ba7cf1729c575f5bbcd6826f2c0#r58088335>
>